### ISSUE TITLE
Unable to access https://www.icloud.com/ at all.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3259]
**Reported by Brendon22 on 2013-06-02 01:41**
Hello,
I am unable to access https://www.icloud.com/ At all with NVDA. After sining in, you are met with a page of buttons that no matter what you do with the keyboard, I am unable to access them. We need to be able to access this website to be able to control our iCcloud on windows. You are also unable to access this website from your computer-Was advised this by apple.

**Comment by derekriemer** [comment]
@jcsteh Is this still the case (I think no).
